node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -1119,6 +1111,67 @@ pub(crate) async fn run<Context>(
 	Ok(())
 }
 
+/// Handle a fetched collation.
+async fn handle_collation_fetched(
```

Comment:
Nit: It's not that we call this function when collation is fetched, but rather when collation fetch result is received. A name and rustdoc update would be good.